src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -214,6 +214,17 @@
             "(?:\\?\\S*)?(?:#\\S*)?",                                      // Useless query or URN appendix
             Pattern.CASE_INSENSITIVE);
 
+    /**
+     * Returns the {@link MessageReference} for this Message. This will be null if this Message has no reference.
+     *
+     * You can access all the information about a reference through this object.
+     * Additionally you can retrieve the referenced Message if discord did not load it in time.
```

Comment:
```suggestion
     * Additionally, you can retrieve the referenced Message if discord did not load it in time. This can be done with {@link MessageReference#resolve()}.
```